homeassistant/components/smhi/config_flow.py

```diff
@@ -0,0 +1,138 @@
+"""Config flow to configure smhi component.
+
+First time the user creates the configuration and
+a location is set in the hass configuration yaml it
+will use that location and create a default
+home location. This behavior will change when
+configuration flow supports default values.
+
+When home location exists a form will show
+to add additional locations for weather data.
+
+The input location will be checked by invoking
+the API. Exception will be thrown if the location
+is not supported by the API (Swedish location only)
+"""
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.const import (CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import aiohttp_client
+from homeassistant.util import slugify
+
+from .const import DOMAIN, HOME_LOCATION_NAME
+
+REQUIREMENTS = ['smhi-pkg==1.0.3']
+
+
+@callback
+def smhi_locations(hass: HomeAssistant):
+    """Return configurations of smhi component."""
```

Comment:
There is no consistency in formatting of smhi between all the documents, they should all be capital

Reply:
That should been taken cared of now. Only one place where i left it with no capital letters, where I reference if user accidentally configure it then it is non capital.